### ISSUE TITLE
activereponse.erb template doesn't wrap xml tags around rules_ids

### ISSUE THREAD

**Issue by MatthewRBruce** [wazuh/ossec-puppet#54]
The activeresponse.erb template doesn't include the xml tags <rules_id> </rules_id> around provided rule ids.  

This results in output that looks like the following:

```
<active-response>
  <command>firewall-drop</command>
  <location>local</location>
  <level>7</level>
  5551<timeout>600</timeout>
  <repeated_offenders>30,60,120</repeated_offenders>
</active-response>
```

**Comment by jlruizmlg** [comment]
hi @MatthewRBruce i just saw your pull request, its to fix this right???

**Comment by MatthewRBruce** [comment]
Yup, Pull Request #56 is to fix this.

**Comment by jlruizmlg** [comment]
Thanks @MatthewRBruce let me take a look and merge!!

**Comment by jlruizmlg** [comment]
Fixed with the pull request #56 thansk @MatthewRBruce
